src/cache.ts

```diff
@@ -1,4 +1,4 @@
-import { ADD, DELETE, ERROR, Informer, ListPromise, ObjectCallback, UPDATE } from './informer';
+import { ADD, CONNECT, DELETE, ERROR, Informer, ListPromise, ObjectCallback, UPDATE } from './informer';
```

Comment:
Does it make sense to add a disconnect here too while we are at it?

Reply:
I think connect + error is sufficient. There shouldn't be a disconnect w/o an error that isn't followed immediately by another connect.